src/sentry/notifications/types.py

```diff
@@ -59,6 +59,9 @@ class NotificationSettingTypes(Enum):
     # Sub category of quotas for warnings before hitting the actual limit
     QUOTA_WARNINGS = 54
 
+    # Sub category of quotas for spend allocation notifications
+    QUOTA_SPEND_ALLOCATIONS = 55
```

Comment:
Just curious
What's the number here indicate?

Reply:
@scefali @leeandher ? would you know?

Reply:
a number that ends in zero appears in `/settings/account/notifications/` while something like 55 can appear if you click `fine tune` on `quota` --> `/settings/account/notifications/quota/`, but i will need to fix the frontend to show it

Reply:
@cathyteng17  So you will need a follow up PR since this is by convention and not really automatic. If you got to `http://localhost:8000/settings/account/notifications/quota/` your new setting doesn't actually render. You'll probably have to add it to https://github.com/getsentry/sentry/blob/master/static/app/views/settings/account/notifications/fields2.tsx :+1: